queue/queue_service.js

```diff
@@ -63,13 +70,16 @@ queueApp.factory("QueueService", function($http){
 				return null;
 			});
 		},
-		finishAppointment: function(time, id, servicedById) {
+		finishAppointment: function(time, id, servicedById, filingStatusIds) {
 			return $http({
 				url: "/server/queue/queue_priv.php",
 				method: 'POST',
-				data: `action=appointmentComplete&time=${time}&id=${id}&servicedById=${servicedById}`,
-				headers: {
-					'Content-Type': "application/x-www-form-urlencoded"
+				params: {
```

Comment:
@CollinsSpencer Btw take a look at this. I ran into an error when I was trying to pass an array back through the url with form-encoded parameters. Did some googling, and it turns out there is this "params" parameter, that enables you to not have to do the form-encoded parameters through the url parameter.

Reply:
Yeah. You can't use the PHP's $_POST though because AngularJS $http defaults Content-Type to application/json, which $_POST doesn't parse. Since queue_priv.php uses $_REQUEST, which does parse application/json formatted parameters, the requests can be made with get and other types of requests in addition to just post requests.

Not a big deal here, but something to be aware of.

Reply:
ah I see, that is good to know.